tools/myst/myst.edl

```diff
@@ -132,7 +134,8 @@ enclave
 
         long myst_nanosleep_ocall(
             [in] const struct timespec* req,
-            [out] struct timespec* rem);
+            [out] struct timespec* rem)
```

Comment:
Does this cause performance improvement? The thread that makes myst_nanosleep_ocall would keep spinning while the host worker thread would actually do the nanosleep.

Reply:
Reverted the change given there is no much benefit

Reply:
If, on the other hand, you do see performance benefit with this change, then it would imply that for really small sleep amounts, it is beneficial to just spin for an approximate equivalent time instead of making an ocall.